source/jormungandr/jormungandr/interfaces/v1/journey_common.py

```diff
@@ -455,6 +455,19 @@ def __init__(self, output_type_serializer):
             hidden=True,
             help="define the duration to reach stop points by crow fly",
         )
+        parser_get.add_argument(
+            "only_ridesharing",
```

Comment:
minor: 
 :gb: `ridesharing_only` would be more correct english-wise

Reply:
the hidden argument should be started with an underscore.